c-sharp-razor/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Base image for the runtime (using Alpine)
+FROM mcr.microsoft.com/dotnet/aspnet:8.0-alpine AS base
```

Comment:
This guy here should be alpine:edge

Reply:
no alpine:edge for C# .Net 👎 ...as far as I know